### PR TITLE
pycairo: Update to the latest release and use the distutils based build system

### PR DESCRIPTION
**Pull request by lazka**

### CODE REVIEW

File: gvsbuild/projects.py [hunk 1]
```diff
@@ -1412,16 +1412,22 @@ def build(self):
...
+            dependencies = ['cairo', 'python'],
+            )
+
+    def build(self):
+        cairo_inc = os.path.join(self.builder.gtk_dir, 'include', 'cairo')
+        old_inc = self.builder.mod_env('INCLUDE', cairo_inc)
+        self.push_location(self.build_dir)
+        self.exec_vs('%(python_dir)s\python.exe setup.py install')
+        self.pop_location()
+        self.builder.restore_env(old_inc)
```
Comment (nacho) [inline gvsbuild/projects.py]: can you also install the license file?

**Comment by lazka** [comment]
Note that this installs pycairo into the Python installation and not into C:\gtk-build\gtk

An alternative which I haven't tried might be to install to C:\gtk-build\gtk\Python36 and extend PYTHONPATH/PYTHONHOME

**Comment by nacho** [comment]
that's fine for now I guess. Thanks

**Comment by lazka** [comment]
ok, thanks!

**Pull request merged**